### PR TITLE
Creating and implementing initial style for login page

### PR DESCRIPTION
**Pull request by casualme2**
We've implemented Login page , the exact style the client wanted.
Also did small fixes (Actually very small fixes) to App and Loader components (removed return because the components , at the time, have only one child)

### CODE REVIEW

File: frontend/src/Components/Loader/Loader.jsx [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-			<div className="text-muted loader-text-size">Loading your data...</div>
+const Loader = () => (
+	<div className="container-fluid d-flex flex-column justify-content-center align-items-center vh-100">
+		<div className="spinner-border text-primary loader-circle-size" role="status">
+			<span className="sr-only">Loading...</span>
 		</div>
-	);
-};
+		<br />
+		<div className="text-muted loader-text-size">Loading your data...</div>
```
Comment (marija-antic) [inline frontend/src/Components/Loader/Loader.jsx:8]: Why having both Loading... and Loading your data...?
Also, break on new line can be done with CSS, but it's no big deal.

File: frontend/src/Components/Login/Login.jsx [hunk 2]
```diff
@@ -1,6 +1,61 @@
 import React from "react";
 import "./Login.css";
+import logoImage from "./assets/logoblackandwhite.svg";
 
-export const Login = () => {
-  return <div>SkyNest</div>;
+const Login = () => {
+	return (
+		<section className="vh-100 container-fluid pageBackgroudColor">
```
Comment (marija-antic) [inline frontend/src/Components/Login/Login.jsx]: This identation seems to big. I use Tabs 2 or Spaces 4

File: frontend/src/Components/Login/Login.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.imageLogo {
```
Comment (marija-antic) [inline frontend/src/Components/Login/Login.css]: I thinnk I saw classes with `-` and now here with camelCase, just choose one style through project.

File: frontend/src/Components/Loader/Loader.jsx (hunk 1 above)
Comment (casualme2) [reply frontend/src/Components/Loader/Loader.jsx:8]: Fixed, thanks.

File: frontend/src/Components/Login/Login.css (hunk 3 above)
Comment (casualme2) [reply frontend/src/Components/Login/Login.css]: We wanted to separate Bootstrap styles with our custom style to know at any point, by just looking at it, if that's our custom style or bootstrap one.
But from now on we'll be consistent with naming them.

Comment (marija-antic) [reply frontend/src/Components/Login/Login.css]: You can come up with a prefix on your custom classes like f.e. SkyNest -> sn-_class_name_. We do sth similar on projetc also.

**Pull request closed without merging**